index.js

```diff
@@ -7,33 +27,221 @@ function ask(questionText) {
   });
 }
 
-// remember the StateMachine lecture
-// https://bootcamp.burlingtoncodeacademy.com/lessons/cs/state-machines
-let states = {
-  'roomOne': { canChangeTo: [ 'roomTwo' ] },
-  'roomTwo': { canChangeTo: [ 'roomThree' ] },
-  'roomThree': { canChangeTo: [ 'roomOne' ] }
+const busStop = new Room();
+  this.roomInv = ["sticks", "bench"]
```

Comment:
`this` is not pointing to the object you just created here, it's pointing to the **global** object